README.md

```diff
@@ -34,10 +34,52 @@ Configure AWS CLI with your credentials as shown here - https://docs.aws.amazon.
 
 `gdk component init --template HelloWorld --language python`
 
-2. Build the artifacts and recipes of the component.
+2. Update configuration in `gdk-config.json`
+   1. Config file `gdk-config.json` would have placeholders:
+   ```json
+    {
+        "component": {
+            "com.example.JavaHelloWorld": {
+                "author": "<PLACEHOLDER_AUTHOR>",
+                "version": "NEXT_PATCH",
+                "build": {
+                    "build_system": "maven"
+                },
+                "publish": {
+                    "bucket": "<PLACEHOLDER_BUCKET>",
+                    "region": "<PLACEHOLDER_REGION>"
+                }
+            }
+        },
+        "gdk_version": "1.0.0"
+    }
+   ```
+   2. Replace `<PLACEHOLDER_AUTHOR>` with your name, `<PLACEHOLDER_BUCKET>` with a s3 bucket name and `<PLACEHOLDER_REGION>` with an aws region.
+   3. After replace these value the `gdk-config.json` file should look similar to:
+   ```json
+    {
+        "component": {
+            "com.example.JavaHelloWorld": {
+                "author": "J. Doe",
+                "version": "NEXT_PATCH",
+                "build": {
+                    "build_system": "maven"
+                },
+                "publish": {
+                    "bucket": "my-s3-bucket",
+                    "region": "us-east-1"
+                }
+            }
+        },
+        "gdk_version": "1.0.0"
+    }
+   ```
+   4. For more details, refer GDK CLI configuration file step in [our documentation](https://docs.aws.amazon.com/greengrass/v2/developerguide/create-components.html#create-component-gdk-cli).
```

Comment:
Move it before or after all the steps.

Reply:
Yeah make sense. Moving it before all the steps.